meshtastic/node_runtime/settings_runtime/__init__.py

```
@@ -0,0 +1,21 @@
+)
+from meshtastic.util import toNodeNum
+
+__all__ = [
+    "_NodeSettingsMessageBuilder",
+    "_NodeSettingsRuntime",
+    "_NodeSettingsResponseRuntime",
+    "_NodeAdminCommandRuntime",
+    "_NodeOwnerProfileRuntime",
+    # For compatibility with code mocking toNodeNum at this module level
+    "toNodeNum",
+]
```

Ruff (0.15.7)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)